mypy/fixup.py

```diff
@@ -185,6 +185,9 @@ def visit_overloaded(self, t: Overloaded) -> None:
         for ct in t.items():
             ct.accept(self)
 
+    def visit_erased_type(self, o: Any) -> None:
+        pass  # Nothing to descend into.
```

Comment:
I think ideally this should raise a `RuntimeError` as `visit_partial_type()` below, because this type only exists temporarily during type inference.